### DISCUSSION TITLE
Auto-Export Report Documentation Button

### DISCUSSION THREAD

**Discussion by DanielBon29** [tabulartools/pbi-explorer#25] [Ideas]
### The feature request

The tool is great! I also suggest the capability to auto-export report documentation.

### Proposed solution

The tool has access to metadata and difficult-to-see features. So, there could be a button to download a predefined excel/PDF document. This would help users to improve the hard proccess of report documentation.

### Additional context

![image](https://github.com/tabulartools/pbi-explorer/assets/87679690/62268de5-1cec-4964-8897-3636a7e24267)

**Comment by DBojsen** [comment]
Hi @DanielBon29 

Thank you for the suggestion.
It makes a lot of sense - it is not on our short term roadmap, but we do have some ideas for exporting both from the inspector and comparer/analyzer.

BR David - Tabular Tools

**Reply by akruzel092615** [reply]
Hello David, 

First off, loving this tool!  Great work!  Being able to export results from the data model explorer to be able transform the data and easily understand how much certain measures/columns are being used and where would be great which I think aligns with this request.  Can you explain why the idea was closed out and moved to a discussion? What does that mean?

Thanks!

Adam
